File: C/day032.c
/* Author: Ahmed A. M. ALOMAR
Date: February 1 - 2020
Description:
This program is an example for Nested If Else Statement
This is a procedural program, and will not take input. */

#include <stdio.h>

int main()
{
  int x = 5;
  int y = 10;
  int z = 15;

  if (x > y && x > z) 
  {
    printf("x is bigger than y and z");
  }
  else if (x < y) 
  {
    if (x < z) 
    {
      printf("z is the biggest number");
    }
  }
  else if (x > z) 
  {
  printf("y is the biggest number");
  }
  else 
  {
  printf("there is something wrong!");
  }
}

File: SourceCode/Python/day032.py
# ------------------------------------ #
# Author: Ahmed A. M. ALOMAR
# Date: May 12 - 2020
#
# Description:
# * Example of Range in python
# ------------------------------------ #


# type
print("\n* The type of the range is: ")
print(type(range(10)))

# One argument
print("\n* Using one argument with range")
for number in range(10):
  print(number)

# Two arguments
print("\n* Using two arguments with range")
for number in range(5, 10):
  print(number)

# Three arguments
print("\n* Using three arguments with range")
for number in range(3, 60, 10):
  print(number)
